WORKSPACE

```diff
@@ -526,7 +526,7 @@ hazel_repositories(
             #   - To build the library : `bazel build @haskell_hlint//:lib`
             # We'll be using it via the library, not the binary.
             hazel_hackage("haskell-src-exts", "1.21.0", "95dac187824edfa23b6a2363880b5e113df8ce4a641e8a0f76e6d45aaa699ff3") +
-            hazel_github_external("ndmitchell", "hlint", "9589299bf9a6273fe7c2915da3f9b7c35f82d527", "49386869b32cf1ccf8241ce9fe927d14475bd693e769ad65670715d3401a959f") +
+            hazel_github_external("shayne-fletcher-da", "hlint", "699adc7c7be5f876433a49661311093c42259cf1", "1fe563fa724ce589b85d67349d98ce254e01cfe28bda8bcf46958d944987e2cb") +
```

Comment:
Should we have a fork under `digital-asset`?

Reply:
Yes, we should.